test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -969,6 +969,91 @@ public void VerifyInitScriptsExecution(ProjectTemplate projectTemplate)
             }
         }
 
+        [DataTestMethod]
+        [DynamicData(nameof(GetNetCoreTemplates), DynamicDataSourceType.Method)]
```

Comment:
Do the template matter in any of these tests?

Reply:
Hi @nkolev92, we are using two different templates since the SDK project having an GitHub issue https://github.com/NuGet/Home/issues/7798 when running the `"get -project"` command, we are using `"GetNetCoreTemplates"` in the other two test cases to cover more templates in the test cases, would you like us to also add `"GetPackagesConfigTemplates"` in other two cases (`VerifyCmdFindPackageExactMatchInPMC` and `VerifyCmdGetPackageUpdateInPMC`?

Reply:
No, I was actually thinking we only need 1 test case. 

The command being run here is not really project specific, so I don't see why we need to get more than 1 template.

Reply:
Hi @nkolev92, we consider that if we test three commands in one case, the others will not run in the test case if the first command fails to execute.

Reply:
I'm not suggesting we test all 3 commands in one case, but rather to remove the project template in this particular test case. 

We don't need `[DynamicData(nameof(GetNetCoreTemplates), DynamicDataSourceType.Method)]`
We only need 1 project of any type.

Reply:
OK, we got it, thanks for your reply, I have remove  remove the project template. Could you please help to review them again. Thanks a lot.